scripts/publish.js

```diff
@@ -35,7 +35,9 @@ execa('lerna', ['ls', '-pl', '--json', '--toposort'])
                     'exec',
                     '--scope',
                     pkg.name,
-                    '--include-dependencies',
+                    pkg.name === '@ovh-ux/manager-react-components'
+                      ? ''
+                      : '--include-dependencies',
```

Comment:
**nitpick:** It's a bad practise to directly inject the package name in a generic `publish` file. Can we handle it through the config on package.json of required library?

Reply:
all we can do its added it in peerDependencies but the cds build failed cause it tried to build the manager react components lib before the apps and it missed the dependancies

Reply:
lerna doesn't resolve correctly the dependencies and tries to build the lib before building its peerDependencies (react-shell-client)